docs/multi-homing.md

```diff
@@ -234,6 +234,67 @@ spec:
 - specifying a static IP address for the pod is only possible when the
   attachment configuration does **not** feature subnets.
 
+## Multi-network Policies
+OVN-Kubernetes implements native support for
+[multi-networkpolicy](https://github.com/k8snetworkplumbingwg/multi-networkpolicy),
+an API providing
+[network policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+features for secondary networks.
+
+To configure pod isolation, the user must:
+- provision a `network-attachment-definition`.
+- provision a `MultiNetworkPolicy` indicating to which secondary networks it
+  applies via the
+  [policy-for](https://github.com/k8snetworkplumbingwg/multi-networkpolicy#policy-for-annotation)
+  annotation.
+
+Please refer to the following example:
+```yaml
+---
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: tenant-blue
+spec:
+    config: '{
+        "cniVersion": "0.4.0",
+        "name": "tenant-blue",
+        "netAttachDefName": "default/tenant-blue",
+        "topology": "layer2",
+        "type": "ovn-k8s-cni-overlay",
+        "subnets": "192.168.100.0/24"
+    }'
+---
+apiVersion: k8s.cni.cncf.io/v1beta1
+kind: MultiNetworkPolicy
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/policy-for: tenant-blue # indicates the net-attach-defs this policy applies to
+  name: allow-ports-same-ns
+spec:
+  podSelector:
+    matchLabels:
+      app: stuff-doer # the policy will **apply** to all pods with this label
+  ingress:
+  - ports:
+    - port: 9000
+      protocol: TCP
+    from:
+    - namespaceSelector:
+        matchLabels:
+          role: trusted # only pods on namespaces with this label will be allowed on port 9000
+  policyTypes:
+    - Ingress
+```
+
+Please note the `MultiNetworkPolicy` has the **exact same** API of the native
+`networking.k8s.io/v1` `NetworkPolicy`object; check its documentation for more
+information.
+
+**Note:** It is currently **required** for the `net-attach-def`s referred to by
+the `k8s.v1.cni.cncf.io/policy-for` annotation to have the `subnet` attribute
```

Comment:
is it `subnet` or `subnets`?
also, let's say something like "`subnets` attribute in its `spec.config` defined". It's probably obvious for more experienced multi-network users where to look for "subnet", but I would question myself if I am looking at the right field :)

Reply:
Done.